tools-local/prebuilt-baseline-online.xml

```diff
@@ -104,6 +104,7 @@
     <Usage Id="Microsoft.Dotnet.WinForms.ProjectTemplates" Version="4.8.1-servicing.19605.5" />
     <Usage Id="Microsoft.DotNet.Wpf.ProjectTemplates" Version="3.0.0" />
     <Usage Id="Microsoft.DotNet.Wpf.ProjectTemplates" Version="3.1.2-servicing.20066.4" />
+    <Usage Id="Microsoft.Extensions.CommandLineUtils" Version="1.0.1" IsDirectDependency="true" />
```

Comment:
Does this mean that we shouldn't remove `common`? (Going by the commit desc "Common has been merged to Extensions", it sounds like Extensions isn't correctly producing this package yet?)

Reply:
It's used online but not offline- there was no offline baseline change, and strangely I didn't have to add it to the `additional-prebuilts-to-delete.txt` either.  I'll look into this a bit more but just based on that it seems safe.